xla/service/gpu/loop_double_buffer_transformer_test.cc

```diff
@@ -470,6 +470,57 @@ ENTRY main {
   // associated computations.
   EXPECT_EQ(while_loops_callees.size(), 8);
 }
+
+TEST_F(GpuLoopDoubleBufferTransformerTest,
+       NestedWhileLoopRemainsFlattenedOddTripCountNestedUnrolled) {
```

Comment:
Name is a bit confusing. Maybe we could rename it to something like `NestedWhileLoopIsAlsoUnrolled`?

Reply:
renamed